File: uikit/src/main/java/screen/groupmemberlist/CometChatGroupMemberListScreenActivity.java
package screen.groupmemberlist;

import android.graphics.Color;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.cometchat.pro.uikit.R;

import constant.StringContract;
import com.cometchat.pro.uikit.Settings.UISettings;

public class CometChatGroupMemberListScreenActivity extends AppCompatActivity {

    private String guid;

    private boolean showModerators;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_block_screen);

        if (getIntent().hasExtra(StringContract.IntentStrings.GUID))
            guid = getIntent().getStringExtra(StringContract.IntentStrings.GUID);
        if (getIntent().hasExtra(StringContract.IntentStrings.SHOW_MODERATORLIST))
            showModerators = getIntent().getBooleanExtra(StringContract.IntentStrings.SHOW_MODERATORLIST,false);

        Fragment fragment = new CometChatGroupMemberListScreen();
        Bundle bundle = new Bundle();
        bundle.putString(StringContract.IntentStrings.GUID,guid);
        bundle.putBoolean(StringContract.IntentStrings.SHOW_MODERATORLIST,showModerators);
        fragment.setArguments(bundle);
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_fragment,fragment).commit();
        if (UISettings.getColor()!=null)
            getWindow().setStatusBarColor(Color.parseColor(UISettings.getColor()));
    }


    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {

           if(item.getItemId()==android.R.id.home){
               onBackPressed();
           }

        return super.onOptionsItemSelected(item);
    }


}

File: uikit-kotlin/src/main/java/screen/groupmemberlist/CometChatGroupMemberListScreenActivity.kt
package screen.groupmemberlist

import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.cometchat.pro.uikit.R
import constant.StringContract

class CometChatGroupMemberListScreenActivity : AppCompatActivity() {
    private var guid: String? = null
    private var showModerators = false
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_block_screen)
        guid = intent.getStringExtra(StringContract.IntentStrings.GUID)
        showModerators = intent.getBooleanExtra(StringContract.IntentStrings.SHOW_MODERATORLIST, showModerators)
        val fragment: Fragment = CometChatGroupMemberListScreen()
        val bundle = Bundle()
        bundle.putString(StringContract.IntentStrings.GUID, guid)
        bundle.putBoolean(StringContract.IntentStrings.SHOW_MODERATORLIST, showModerators)
        fragment.arguments = bundle
        supportFragmentManager.beginTransaction().replace(R.id.frame_fragment, fragment).commit()
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            onBackPressed()
        }
        return super.onOptionsItemSelected(item)
    }
}